controllers/promise_controller.go

```diff
@@ -938,12 +938,14 @@ func setStatusFieldsOnCRD(rrCRD *apiextensionsv1.CustomResourceDefinition) {
 			Status: &apiextensionsv1.CustomResourceSubresourceStatus{},
 		}
 
-		rrCRD.Spec.Versions[i].AdditionalPrinterColumns = []apiextensionsv1.CustomResourceColumnDefinition{
-			{
-				Name:     "status",
-				Type:     "string",
-				JSONPath: ".status.message",
-			},
+		if len(rrCRD.Spec.Versions[i].AdditionalPrinterColumns) == 0 {
```

Comment:
Is there an expected behaviour here about merging things? As in, if the user does not set status in their additional columns, should we keep status? I think this was the only other thing discussed a while back.

Reply:
> caveat: i don't remember being in a conversation about this

i thought about merging, but decided against it. my reasoning: 
* not all users will set the columns; for those, adding the `.status.message` is the standard Kratix behaviour
* the users that define the additional columns should have _full_ control of what's displayed
  * that means, if we merge by default, i'd say we would need to provide a way for it to be hidden 
  * with the replace approach, they can always add the `status.message` column by adding it themselves to the list

Reply:
merging as is for now, we can always change it if users find it too confusing